lib/rucio/core/permission/generic.py

```diff
@@ -722,6 +749,17 @@ def perm_get_request_by_did(issuer, kwargs):
     return True
 
 
+def perm_get_request_history_by_did(issuer, kwargs):
+    """
+    Checks if an account can get a historical request by DID.
+
+    :param issuer: Account identifier which issues the command.
+    :param kwargs: List of arguments for the action.
+    :returns: True if account is allowed, otherwise False
+    """
+    return True
```

Comment:
I wouldn't allow anybody to query an endpoint which can DDOS rucio.

Reply:
Limited permissions to this.